src/client/game/game-reducer.js

```diff
@@ -127,24 +135,58 @@ const game = (state = initialState, action) => {
                 submission: false,
                 roundResults: false,
                 finalAnswers: [],
-                resetRound: true
-            })
+                resetRound: true,
+                randomNumber: null,
+                numberList: []
+            });
         case actionTypes.RESET_FINISHED: {
             return updateState(state, {
                 resetRound: false,
                 disableConsonant: false,
-                disableVowel: false
-            })
+                disableVowel: false,
+                disableLarge: false,
+                disableSmall: false
+            });
         }
         case actionTypes.GAME_PARAMETERS:{
             return updateState(state,{
                 gameParams: action.payload
-            })
+            });
         }
         case actionTypes.GAME_FINISHED: {
             return updateState(state, {
                 gameFinished: true
-            })
+            });
+        }
+        case actionTypes.NEXT_ROUND_TYPE: {
```

Comment:
This file should have tests too eventually. One of the good things about unit tests is that it documents the code so it's easier to understand. This is one of the reasons to have tests by the code (in the same folder).